docs/guide/options/README.md

```diff
@@ -204,23 +209,85 @@ export const mutations = {
+The option further adds a serverMiddleware that checks on server side if the token is valid and then injects a simplified [`admin.auth.UserRecord`](https://firebase.google.com/docs/reference/admin/node/admin.auth.UserRecord) into the context variable `res.locals.user`.
+
+The simplified user record contains the following properties:
+
+- `uid`: The users uid
+- `email`: The users email
+- `emailVerified`: If the email was verified
+- `displayName`: The users display name
+- `allClaims`: All claims from the [admin.auth.DecodedIdToken](https://firebase.google.com/docs/reference/admin/node/admin.auth.DecodedIdToken)
+
+The injected user can be used by context aware life cycle hooks on the server side (e.g. the store action `nuxtServerInit`).
 
 A tutorial on how to set this up can be found [here](/tutorials/ssr/#firebase-auth-in-ssr-universal-mode).
 
+##### Firebase admin authorization
+
+If you want additional information on the user the module can inject a full [`admin.auth.UserRecord`](https://firebase.google.com/docs/reference/admin/node/admin.auth.UserRecord) into the `ctx.res.locals.user` property.
+
+The `allClaims` property is set in addition to the default user record properties.
+
+To enable this you can authorize the firebase admin by [generating a service account key](https://firebase.google.com/docs/admin/setup#initialize-sdk) and linking to it with the following configuration:
+
+```js
+auth: {
+  ssr: {
+    // this is required if ssr is an object
+    credential: '/absolute/path/to/serviceAccount.json'
+
+    // nuxt aliases are supported
+    credential: '~/assets/serviceAccount.json'
+
+    // if this is set to true the credential will be retrieved from GOOGLE_APPLICATION_CREDENTIALS environment variable
+    credential: true
+
+    // The service worker session automatically ignores external resources, static files and HMR calls
+    // If you need to ignore additional routes, define them here
+    ignorePaths: [
+      '/admin', // path is ignored if url.pathname.startsWith('/admin')
+      /^api/ // path is ignored if url.pathname without the leading slash (/) matches the RegExp
+    ]
+  }
+}
+```
+
+##### Server side Firebase client SDK login
+
+Once you have [properly setup the admin sdk](#firebase-admin-authorization) you can enable server side login to use firebase services on the server, e.g. to perform store hydration on page load.
+
+Simply set `auth.ssr.serverLogin = true`.
+
+::: warning Do not use firebase client SDK to perform API operations.  
+
+Server is stateless and has to sign in on every request.  
+Firebase client SDK login attempts are rate limited by IP, multiple server side calls will cause an 'auth/too-many-requests' error.
+
+If you have an API which is served over nuxt ssr, add it's prefix to `auth.ssr.ignorePaths`
```

Comment:
is this sentence complete?

Reply:
I improved the description and added an example (to be committed)

Reply:
I'm currently waiting for response from firebase community and have not yet implemented load tests.
Once I have a clearer understanding of the issue I'll update the warning accordingly